nova/conf/vmware.py

```diff
@@ -488,6 +488,21 @@
 that are stored in Swift will be downloaded by VMWare from the `direct_url`,
 instead of the nova-compute service having to proxy the image between glance
 and VMware.
+"""),
+    cfg.IntOpt('resource_disk_gb_max_unit_limit',
+               min=0,
+               default=0,
```

Comment:
I would go with `-1` for unlimited, because we could also disable local disks, and then we might want to set the value to `0` to report no local disks, and often negative values are taken as unlimited.

Reply:
I get that point and thought about it, but wasn't sure if `0` makes sense.

Checking [Placement](https://github.com/sapcc/placement/blob/17cb78c072b2cd7d10b59194b0f167629858fda9/placement/schemas/inventory.py#L41-L45), it doesn't allow `0` to be set as `max_unit`. Therefore, `0` would not be a valid value. We would have to add extra code to explicitly disallow `0` if we don't use it as "unlimited".

Reply:
I don't think extra code is needed:

`resource_disk_gb_max_unit_limit = 0` [results here](https://github.com/sapcc/nova/pull/428/files/fbdf5b78be0e3d8395c6b84762093d8d6f7b0720#diff-0e23ae0d8359d42642027d6ae0bc438cb64626947decc427afa47a5430737b20R553-R555) `local_gb_max_free==0`, so no resource of the type `DISK_GB` will be reported.

I assume here, no local disk is supported at that is the way to report it. I haven't verified it though.

Reply:
Ah, nice. Then we can change it to `-1`, sure. :)